pkg/lib/size/size_test.go

```diff
@@ -0,0 +1,55 @@
+package size
+
+import (
+	"testing"
+)
+
+func TestSize(t *testing.T) {
+	type args struct {
+		endpoint  string
+		accessKey string
+		secKey	  string
+	}
+	tests := []struct {
+		name  string
+		args  args
+		low   int64 // lowest acceptable size due to variations in iso hdrs
+		exact int64
+		high  int64 // highest acceptable size due to variations in iso hdrs
+	}{
+		{
+			name:  "small tinyCore.iso file size",
+			args:  args{"file:///root/go/src/kubevirt.io/containerized-data-importer/tinyCore.iso", "", ""},
+			low:   17860000,
+			exact: 18874368,
+			high:  18900000,
+		},
+		{
+			name:  "large windows .iso.xz file size",
+			args:  args{"file:///root/go/src/kubevirt.io/containerized-data-importer/en_windows_server_2016_updated_feb_2018_x64_dvd_11636692.iso.xz", "", ""},
```

Comment:
where does this file come from?  Also, if it's being downloaded at runtime, why into the repo root?

Reply:
same debugging related mistake